### ISSUE TITLE
adapt-contrib-vanilla: ensuring contrast when using background image functionality

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt_framework#2828]
A common issue when using Vanilla background image config is ensuring good contrast between the content and the background image. Many of us will be use to editing and preparing imagery before applying to a course but not everyone has experience/access to image editing software. 

I think including some contrast options that can be combined with background image functionality would be of use, particularly for AAT users. Transparency or gradient overlays that can darken or lighten an image would provide contrast and could be applied at all levels (header/article/block). These could be implemented as pre-bundled classes or as config. I think classes would be simpler and could be expanded upon in the theme as well as creating some consistencies when applying in a course but happy to take steer from others.

**Comment by guywillis** [comment]
Agreed that classes would likely be simpler to implement and use across both Framework and Authoring Tool. 

It's likely we'd have to reduce the number of potential gradient overlays that are offered to due the overwhelming number of types and variants that can be applied in CSS. Even limiting the gradients to linear, two colour, and 4 directions creates 8 classes though I think this provides a good base to build from should it be required.
